src/query.rs

```diff
@@ -51,7 +53,17 @@ impl From<bool> for FilterValue {
     }
 }
 
-/// Build an SQL query for the given filters, over the given type (using it's name and fields).
+impl FilterValue {
+    pub fn IsA<'de, T>() -> WMIResult<Self>
```

Comment:
Maybe copy-paste a part of the example in `lib.rs` as a docstring for this? (No need to actually query anything, just a simplified example)